controls/radwindow/how-to/hide-minimize-maximize-buttons.md

```diff
@@ -35,7 +35,33 @@ The easiest way to hide the minimize or maximize buttons is to set the __HideMin
 
 ## Use the ResizeMode property
 
-By setting the __ResizeMode__ property to __CanMinimize__ only the __Minimize__ button will be visible. The resize button will get hidden and the resizing will get disabled. By setting the __ResizeMode__ property to __NoResize__ both the Minimize and the Maximize buttons will get hidden, the resizing will also get disabled. 
+By setting the __ResizeMode__ property to __CanMinimize__ only the __Minimize__ button will be visible. The resize button will get hidden and the resizing will get disabled. This is demonstrated in __Example 2__.

```

Comment:
I think that the resize button will get hidden should be "the __maximize__ button will __be__ hidden". 

Also get hidden and get disabled do not sound right. We can change them to __be__ hidden and __be__ disabled.